### ISSUE TITLE
Traceback form user

### ISSUE THREAD

**Issue by schriftgestalt** [jenskutilek/UnicodeInfo-Glyphs#5]
Traceback (most recent call last):

  File "plugin.py", line 9, in <module>
    from jkUnicode.aglfn import getGlyphnameForUnicode, getUnicodeForGlyphname

ModuleNotFoundError: No module named 'jkUnicode.aglfn'

App:3.2.3-3260 Plugin:1.0.3-7 System:14.1.2

**Comment by jenskutilek** [comment]
The jkUnicode module should now be installed as a dependency in the plugin manager.

Hm, now that I think about it, probably the dependencies are only installed on first install, not when the plugin is updated?

**Comment by jenskutilek** [comment]
I tried another branch with logic to install the module via pip if it is missing:

https://github.com/jenskutilek/UnicodeInfo-Glyphs/blob/install-dependencies/UnicodeInfo.glyphsPlugin/Contents/Resources/installHelper.py

But that brings other problems. I've written about my experience here: https://forum.glyphsapp.com/t/python-versions-not-finding-glyphs-python-version-glyphsapp-objc/20640/24

For now, uninstalling UnicodeInfo and installing it again through the Plugin Manager should fix the problem.

**Comment by schriftgestalt** [comment]
Did you add a check and an error message when the module is not there?

**Comment by jenskutilek** [comment]
No, I thought that the plugin manager installing the depencency would be enough. Should I add a separate check?

**Comment by schriftgestalt** [comment]
But when the module is missing it is just crashing. It should show a message how to fix it.

**Comment by jenskutilek** [comment]
I've added the message.
